_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
 
 ## The Notification is dead, long live the Notification!
 
-If you care about such things, you have likely noticed that NewPipe's notification looks quite different from most media player apps these days. It has a progress bar, buttons in odd places, and a permanent grey colour. This is because NewPipe uses an old API for notifications called RemoteView. This allowed for a lot of customisation, and that was good at the time it was in vogue. However, it has been deprecated and superseded by a more modern API.
+As a pedant, you have noticed NewPipe's notification is different from most other media apps. It has the bar of progress, buttons in odd places, and a permanent, though tepid grey colour. Much of the difference is down to using an old API for notifications called RemoteView, _in the past_!
 
 Enter the **Age of MediaStyle notification**:
 
-This is a special notification type which has been available for media apps to use since Android Lollipop. In Oreo, it gained the ability to be coloured to match the colour theme of the cover art of the music playing. And since Android 10, it has a seekbar built right into it! It is a powerful API that removes the need to use a whole bunch of custom code to achieve what we had so far. This will make it far easier to maintain and improve NewPipe's notification. Win for development! So let's get on with what is new and what has changed.
+This is a special notification type available for media apps since Android (5.0) Lollipop. In Oreo (8.0), it could colour-match its theme to the cover art of the music playing. Since Android 10, it has a SeekBar built in. It is a powerful API removing the need to use a whole bunch of custom code to achieve what was. Easier to maintain and improve. Win!
 
 ### Android 9
 
-First of all, we say "sorry" to our users on Android 9 and below. The progress bar is gone. As we've said above, it will only be shown on Android 10 and above. We're a small team and can't maintain code for the old and new types of notifications simultaneously, so the old one has to go. And not just that; the old notification isn't even supported Android 11 onwards. Bye-bye RemoteView notifications. We shall miss you! :'(
-
-But you get a lot in exchange for that sacrifice!
+Android 9 is not Android 10, and there is now only a progress bar on 10. It makes sense to keep things managable by our small team. The old notification isn't even supported on Android 11 onwards. RemoteView notifications are out, MediaStyle is in.
+Your sacrifice is not in vain:
 
 ### New features
 
-You get five CUSTOMISABLE - you read it right! - **CUSTOMISABLE** action buttons to control playback. There is a new <code><img src="/img/icons/play_arrow-24dp.svg" /> notification</code> category in the app settings, and you can choose which button should perform which player action. Play/pause, forward/rewind, previous/next, shuffle, repeat, close, or even nothing!
```

Comment:
Now this is a sales-pitch. With CAPITAL LETTERS. That isn't done in the strings, and it shouldn't be done anywhere. It is shouting, and then it is shouting in bold text. I have buttons and I control playback, and I control playback, and what does playback mean in terms of buttons. Oh man.

Reply:
> Now this is a sales-pitch.

It is meant to read like one. That is why we went completely over the top with it. That _is_ the joke.

> That isn't done in the strings

Again, we'll address this after your explanation of "the strings".

> It is shouting, and then it is shouting in bold text.

Sales pitch tone. Intentional for joke.

> I have buttons and I control playback, and I control playback, and what does playback mean in terms of buttons. Oh man.

Yes, standard sales pitch stuff. Over-emphasising what doesn't need to be over-emphasised.

Reply:
There is no distance established though, it just mimics the unwanted.